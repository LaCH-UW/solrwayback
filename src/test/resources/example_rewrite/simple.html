<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index_o1.css" />
</head>
<h1>A very simple page</h1>

<p>An <a href="http://example.com">absolute link</a> leading to example.com</p>
<p>A <a href="subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</tt></p>
<p>A <a href="/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>

<p>An absolute referenced image: <img src="http://example.com/image_o2.jpg"/></p>
<p>A relatively referenced image: <img src="subfolder/image_o3.jpg"/></p>
<p>A root referenced image (same as the absolute one): <img src="/image_o2.jpg"/></p>

</html>
