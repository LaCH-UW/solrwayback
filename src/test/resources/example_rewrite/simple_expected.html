<html>
 <head>
  <title>Some page</title>
  <link rel="stylesheet" type="text/css" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=1">
 </head>
 <body>
  <h1>A very simple page</h1>
  <p>An <a href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com">absolute link</a> leading to example.com</p>
  <p>A <a href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/somefolder/subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</p>
  <p>A <a href="http://localhost:0000/solrwayback/services/web/2020043030700/http://example.com/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>
  <p>An absolute referenced image: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2"></p>
  <p>A relatively referenced image: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=3"></p>
  <p>A root referenced image (same as the absolute one): <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2"></p>
 </body>
</html>